### DISCUSSION TITLE
Non-Empty date field just ignores the date input with dateFormat: 'm-d-Y'

### DISCUSSION THREAD

**Discussion by stevengpn** [pocketbase/pocketbase#2716] [Q&A]
It happens when I used  **dateFormat: 'm-d-Y'**, by below code that overwrites the default one in CommenHelper.js
It keeps complaining missing required input value when click save button of the form (RecordUpsertPanel)
It works fine once switched back to 'Y-m-d'
`<Flatpickr
        id={uniqueId}
        options={{...CommonHelper.defaultFlatpickrOptions(),
                  enableTime: false,
                  dateFormat: 'm-d-Y'
                }}
        placeholder={suggest_value}
        bind:value={pickerValue}
        bind:formattedValue={value}
        on:close={onClose}
    />`

**Comment by ganigeorgiev** [comment]
I don't follow. Please elaborate what you are trying to do.

Note that PocketBase dates at the moment are always stored in the format `Y-m-d H:i:s.uZ` (the ms and zone part are trimmed on the frontend when passed to the date field).

**Reply by stevengpn** [reply]
Cheers! How can I make it in format of US: 'm-d-Y' showing in the UI? ideally with no time string followed.

**Reply by ganigeorgiev** [reply]
The Admin UI at the moment is not extendable or customizable.

If you want to change it in your own fork you'll have to reformat the datetime string value before passing it to the calendar component. We have a `luxon` helper that you may use:

```js
// note that luxon uses a different date format
// https://moment.github.io/luxon/#/parsing?id=table-of-tokens
CommonHelper.formatToUTCDate(value, "MM-dd-yyyy")
```
